text/0107-construct-library-module-lifecycle.md

```diff
@@ -30,20 +30,27 @@ module:
 - Stage 2 - Developer Preview
 - Stage 3 - GA
 
+Each stage in this document describes 3 distinct sets of activities:
+
+* **Entry activities:** Actions that need to be taken before the module enters this stage. These can either be automatic, or done by the team. They will usually be performed once a module is deemed for stage maturation, and all its exit criteria are met.
```

Comment:
This is weird. One of our linter rules is that line length should not exceed 150 - https://github.com/aws/aws-cdk-rfcs/blob/master/tools/linters/markdownlint.json#L6-L8.

Wonder why this is not honoured. While I investigate this, can you fix this up? And in other places in the doc.

Reply:
In the mean time I fix this, please run the commands on here https://github.com/aws/aws-cdk-rfcs/blob/master/.github/workflows/pr-lint.yml and fix all violations.

Reply:
I found the issue and fixed it up here - https://github.com/aws/aws-cdk-rfcs/pull/136

Sorry :(

Reply:
👍